arch/arm/boot/dts/imx6ul-ts7100-5.dts

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0+ or X11
+/*
+ * Copyright (C) 2023 Technologic Systems, Inc. dba embeddedTS
+ */
+
+/dts-v1/;
+#include "imx6ul-ts7100.dtsi"
+#include "imx6ul-ts7100-5.dtsi"
+
+/ {
+	model = "embeddedTS i.MX6UL TS-7100 (5)";
```

Comment:
I'm okay with this, I just want to make sure that the `TS-7100 (5)` format isn't going to be an issue in the production process.

Reply:
No worries there, its a bit undefined right now on how we'll use this. We do something similar where the baseboard definitions for the 4900 are in parenthesis like this as well.  If we have a more friendly name for boards in the future rather than their ID, we'll use that instead.
https://github.com/embeddedTS/linux-lts/blob/linux-5.10.y/arch/arm/boot/dts/imx6q-ts4900-2.dts#L12C6-L12C6